go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/jenkins-x/jx-kube-client/v3 v3.0.2
 	github.com/jenkins-x/jx-logging/v3 v3.0.6
 	github.com/jenkins-x/lighthouse-client v0.0.295
+	github.com/keskad/jx-helpers/v3 v3.0.67-0.20220312071418-f6abb1196975  // todo: revert before merging and bump to latest version
```

Comment:
Before merging please merge https://github.com/jenkins-x/jx-helpers/pull/362 , then switch there back to JX repository and bump a version.